store/variable.go

```diff
@@ -36,6 +38,7 @@ NewVar stores a new variable in a Store.
 	x := store.NewVar(s, 10) // x is stored in s.
 */
 func NewVar[T any](s Store, data T) Var[T] {
+	connect.Connect(con, &newVarFunc)
```

Comment:
why do we need to connect here now and before not?

Reply:
Before every call to `connect()` would connect all components. Well only the first call would connect them all and all subsequent calls would be noops.